File: csharp/sources/Valkey.Glide.InterOp/Errors/RequestException.cs
namespace Valkey.Glide.InterOp.Errors;

/// <summary>
/// An error on Valkey service-side that was reported during a request.
/// </summary>
public sealed class RequestException : GlideException
{
    public RequestException() : base() { }

    public RequestException(string message) : base(message) { }

    public RequestException(string message, Exception innerException) : base(message, innerException) { }
}

File: JNI-java/client/src/main/java/glide/api/models/exceptions/RequestException.java
/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
package glide.api.models.exceptions;

/** Request error: Errors that were reported during a request. */
public class RequestException extends GlideException {
    public RequestException(String message) {
        super(message);
    }
}
